roles/openshift_aws/defaults/main.yml

```diff
@@ -333,6 +327,20 @@ openshift_aws_launch_config_security_groups:
   - "{{ openshift_aws_clusterid }}_infra"  # node type sg
   - "{{ openshift_aws_clusterid }}_infra_k8s"  # node type sg k8s
 
+openshift_aws_launch_config_iam_roles:
+  master:
+    name: "{{ openshift_aws_clusterid }}-iam_master"
```

Comment:
name: "{{ openshift_aws_iam_master_role_name | default(openshift_aws_clusterid ~ '-iam_master') }}"

Do this for the others as well.